File: src/it/nominasuntsubstantiarerum/netbus/entity/EntityCitta.java
package it.nominasuntsubstantiarerum.netbus.entity;

public class EntityCitta {
	private int idCitta;
	private String nome;
	
	public EntityCitta(String nome) {
		this.nome = nome;
	}
	
	public EntityCitta(int idCitta, String nome) {
		this.idCitta = idCitta;
		this.nome = nome;
	}
	
	public int getIdCitta() {
		return idCitta;
	}
	
	public void setIdCitta(int idCitta) {
		this.idCitta = idCitta;
	}
	
	public String getNome() {
		return nome;
	}
	
	public void setNome(String nome) {
		this.nome = nome;
	}
}

File: htdocs/entity/EntityCitta.php
<?php

class EntityCitta {
    private int $idCitta;
    private String $nome;

    public function __construct(int $idCitta, String $nome) {
        $this->idCitta = $idCitta;
        $this->nome = $nome;
    }

    // Getter e Setter per idCitta
    public function getIdCitta(): int {
        return $this->idCitta;
    }

    public function setIdCitta(int $idCitta): void {
        $this->idCitta = $idCitta;
    }

    // Getter e Setter per nome
    public function getNome(): String {
        return $this->nome;
    }

    public function setNome(String $nome): void {
        $this->nome = $nome;
    }
}
?>